nautobot/extras/jobs.py

```diff
@@ -433,7 +433,7 @@ def deserialize_data(cls, data):
 
         return return_data
 
-    def validate_data(self, data):
+    def validate_data(self, data, files=None):
```

Comment:
`validate_data` is only being used for the API and just seems to be an abstraction for running `.as_form(...).is_valid()` which the UI does directly.

Don't know if this abstraction is necessary or if we should further collapse the logic the UI views do into a function both it and the API can use directly.

Reply:
We should further collapse the logic. We have this problem in more than one place with Jobs. 😭 

I'm not saying that this is the time to do it, but we should address these as a part of the Jobs overhaul at minimum, unless you think you can reasonable tackle it now, however, it might not be worth it considering we're approaching v2 dev.